Fixed Streaming Category (Maybe?)

NiceHacksDude opened the pull request:
Fixed Streaming Category

Changes: 
- Added /u/nbatman's list.
- Changed /u/nbatman link to remove reddit link since comment is deleted, we could always just remove it all together since the list has been transferred over.
- Removed some sites that I knew off the top of my head that have shut down (pron.tv/alluc.ee)
- Added Alternate Sites (Alluc.uno)
- Added Subcategories that match /u/nbatman's
- Temporarily moved all other previous streaming sites in uncategorized category, will put in subcategories once I test each site, new sites added could go in this category or until we figure out the correct subcategory for it.

Important:
- Haven't checked yet if all sites work.
- Can't see any typos from my view, let me know if anyone sees anything.
- Let me know what I should change if you don't like this current change.

NiceHacksDude commented:
Definitely will continue work on sub categories and giving more descriptions on sites/removing sites that no longer work, adding new ones, etc. Let me know if this is a step in the right direction.

Igglybuff commented:
Sounds great, thanks for this. I'll take a closer look at the diff when I'm not on mobile :)

NiceHacksDude commented:
More Changes: 
Changes: 
- Moved some sites into correct categories. 
- Moved Sporting Streams into subcategory for streaming sites.
- Added Best Streaming Services and Best Sport Streaming Services (Opinion rating for sites)
- Added few descriptions
- Changed Openload to third party hosts to include streammango/other sites.

To Do: 
- Test Sites
- Work on categories, TV and HD Streaming overlap a lot.
